lib/active_model/serializer/lazy_association.rb

```diff
@@ -11,7 +11,8 @@ def reflection_options
       end
 
       def object
-        @object ||= reflection.value(
+        return @object if defined?(@object)
+        @object = reflection.value(
```

Comment:
looks fine to me. I write code like this all the time.

mind adding an entry to the changelog under fixes?

Reply:
done